core/trino-main/src/main/java/io/trino/sql/planner/plan/TableWriterNode.java

```diff
@@ -437,22 +436,17 @@ public boolean supportsMultipleWritersPerPartition(Metadata metadata, Session se
     public static class RefreshMaterializedViewReference
             extends WriterTarget
     {
-        private final Table table;
+        private final String table;
```

Comment:
I think we should name this variable `tableToStringRepresentation` to stress that it is only used for `toString()` method. Otherwise one could try to use it to something else in future.

Another idea is to remove this field completely and maybe use `TableHandle` in `toString()`

Reply:
I think it's obvious that a raw name shouldn't be used for something else. Do you have something in mind where this would be used and the author wouldn't quickly discover that it's wrong?